src/components/Common/AvatarEditModal.tsx

```diff
@@ -387,24 +530,43 @@ const AvatarEditModal = ({
                 <div className="flex flex-1 items-center justify-center">
                   {!previewImage ? (
                     <>
-                      <Webcam
-                        audio={false}
-                        height={720}
-                        screenshotFormat="image/jpeg"
-                        width={1280}
-                        ref={webRef}
-                        videoConstraints={constraint}
-                        onUserMediaError={async () => {
-                          const requestValue = await requestPermission("user");
-                          if (!requestValue.hasPermission) {
-                            setIsCameraOpen(false);
-                          }
-                        }}
-                      />
+                      <div className="relative aspect-square max-w-[720px] w-full overflow-hidden">
+                        <Webcam
+                          audio={false}
+                          height={720}
+                          width={720}
```

Comment:
Webcam shows 720x720 in the initial preview (when cam is on, before capturing image). But once image is captured and we switch to crop mode, it's clear that 1280 width image was captured which would be confusing to the user.